Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -314,16 +302,19 @@ open class NavigationMapView: UIView {
             
             let offset = (route.distance - routeAlternative.infoFromDeviationPoint.distance) / route.distance
             parentLayerIdentifier = addRouteLayer(route,
-                                                  fractionTraveled: offset,
                                                   below: parentLayerIdentifier,
-                                                  reuseExistingLayer: true,
                                                   isMainRoute: false,
                                                   legIndex: nil)
+            if let altertiveRouteLayerIdentifier = parentLayerIdentifier {
```

Comment:
Did you mean `alternativeRouteLayerIdentifier` in this case?

Reply:
Thanks for the suggestion, I'll update the name in https://github.com/mapbox/mapbox-navigation-ios/pull/4015